packages/wonder-stuff-ci/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# @khanacademy/wonder-stuff-ci
+
+## 0.0.1
+
+- Initial commit of porting over OLC functions for mobile release management
```

Comment:
How should I format the change log in wonderstuff? Is there a getting started guide on creating a package in wonder-stuff? All I did was copy what another package was doing.

Reply:
Ahh no - so we're using Changesets! The CHANGELOG is generated automatically, so you shouldn't make it. You can see how to do this here: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2217640195/Versioning+and+releasing+Wonder+Blocks+packages (it's Wonder Blocks but we follow the same process for Wonder Stuff)